### ISSUE TITLE
Feature: Add custom containerClass for ngx-bootstrap

### ISSUE THREAD

**Issue by benj0c** [isaacplmann/ngx-tour#150]
It helps when you want to add custom style only for tour step popovers.
I've already create PR for that #149 

What do you think? Make sense?

**Comment by isaacplmann** [comment]
Can you solve this by creating your own custom template?  I'm leery of adding more configuration options if it's possible to do with the existing options.

**Comment by benj0c** [comment]
I know for custom template, but how can I manage the container popover element? I wanna style popover container (not popover-body) element and that isn't possible with custom template. With custom template you can only put some elements inside popover-body div element. Or I'm wrong?

One thing also, if I somehow do it with custom templates, I can't make for example three different popover's style, because custom template is one and only (per <tour-step-template>).

**Comment by isaacplmann** [comment]
(1) You're right about not being able to add a class to the popover container from the template.

(2) There's only one template, but you can dynamically set the class in your template, like this:
```html
<tour-step-template>
  <ng-template let-step="step">
    <div class="wrapper" [ngClass]="step.customStepClass">
    <p class="tour-step-content">{{step?.content}}</p>
    <div class="tour-step-navigation">
      <button *ngIf="tourService.hasPrev(step)" class="btn btn-sm btn-default" (click)="tourService.prev()">« {{step?.prevBtnTitle}}</button>
      <button *ngIf="tourService.hasNext(step)" class="btn btn-sm btn-default" (click)="tourService.next()">{{step?.nextBtnTitle}} »</button>
      <button class="btn btn-sm btn-default" (click)="tourService.end()">{{step?.endBtnTitle}}</button>
    </div>
    </div>
  </ng-template>
</tour-step-template>
```

**Comment by isaacplmann** [comment]
Do you need to modify the arrow or title portion of the popover?

**Comment by benj0c** [comment]
I need to modify style of root element of following snippet:

`<popover-container ... class="popover ....">
  ....
  <tour-step-template>
    <ng-template let-step="step">
      <div class="wrapper" [ngClass]="step.customStepClass">
      <p class="tour-step-content">{{step?.content}}</p>
      <div class="tour-step-navigation">
        <button *ngIf="tourService.hasPrev(step)" class="btn btn-sm btn-default" (click)="tourService.prev()">« {{step?.prevBtnTitle}}</button>
        <button *ngIf="tourService.hasNext(step)" class="btn btn-sm btn-default" (click)="tourService.next()">{{step?.nextBtnTitle}} »</button>
        <button class="btn btn-sm btn-default" (click)="tourService.end()">{{step?.endBtnTitle}}</button>
      </div>
      </div>
    </ng-template>
  </tour-step-template>
</popover-container>`

**Comment by isaacplmann** [comment]
You convinced me.  Merged and released as 4.1.0.

**Comment by lincanli** [comment]
> You convinced me. Merged and released as 4.1.0.

Hi, just wandering when will 4.1.0 be released

**Comment by isaacplmann** [comment]
https://www.npmjs.com/package/ngx-tour-ngx-bootstrap
